src/EnergyPlus/ScheduleManager.cc

```diff
@@ -226,6 +227,7 @@ namespace ScheduleManager {
 		using DataSystemVariables::iUnicode_end;
 		using DataSystemVariables::TempFullFileName;
 		using DataSystemVariables::CheckForActualFileName;
+		using DataSystemVariables::UseImportedSunlitFrac;
```

Comment:
This isn't being used in this function and is throwing the build warning.  That will clean up CI some.

Reply:
Cleaned.